ci/openshiftio-appdev-docs/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+launcher.template_url.latest=https\://raw.githubusercontent.com/openshiftio/launchpad-templates/v5/openshift/launchpad-template.yaml
```

Comment:
FYI @tradej you had a regression on the link and i fixed it. It was going to v4 instead of v5. Other than that, this LGTM